core/kernel/console.c

```diff
@@ -57,6 +58,17 @@ void register_serial_console(struct serial_chip *chip)
 	serial_console = chip;
 }
 
+#ifdef CFG_CONSOLE_RUNTIME_DISABLE
+static TEE_Result console_runtime_disable(void)
+{
+	/* Do not print trace messages after this point */
+	trace_set_level(TRACE_MIN);
+
+	return TEE_SUCCESS;
+}
+boot_final(console_runtime_disable);
```

Comment:
CI checkpatch complains on a missing empty line above. Could you add it?
CI failure on QEMUv8/Hafnium seems unrelated (network issues?)